utils/frame/benchmarking-cli/src/storage/record.rs

```diff
@@ -107,6 +107,7 @@ impl BenchRecord {
 	}
 }
 
+// TODO refactor once https://doc.rust-lang.org/test/stats/trait.Stats.html is stabilized.
```

Comment:
for TODOs, please open an issue, and put the issue number into the commented code

Reply:
Looking at this again: it says `This is a nightly-only experimental API. (test)`.  
Does this mean that it is only available for testing? Then I cannot use it...